packages/widgets/src/FieldSetField/FieldSetField.js

```diff
@@ -0,0 +1,81 @@
+import propTypes from '@dhis2/prop-types'
+import React from 'react'
+
+import { sharedPropTypes } from '@dhis2/ui-constants'
+import { Field, FieldSet, Legend } from '@dhis2/ui-core'
+
+/**
+ * @module
+ * @param {FieldSetField.PropTypes} props
+ * @returns {React.Component}
+ *
+ * @example import { FieldSetField } from '@dhis2/ui'
+ *
+ * @example import { FieldSetField } from '@dhis2/ui-widgets'
+ */
+const FieldSetField = ({
+    children,
+    className,
+    disabled,
+    helpText,
+    validationText,
+    legendText,
+    required,
+    dataTest,
+    valid,
+    error,
+    warning,
+}) => (
+    <FieldSet classname={className} dataTest={dataTest}>
+        {legendText && <Legend required={required}>{legendText}</Legend>}
+
+        <Field
+            disabled={disabled}
+            required={required}
```

Comment:
I think we can safely omit these, since we're not passing a label, so neither of these will be shown. Instead, should we have a way to show disabled status on a Legend?

Reply:
086398639dff766dbeddc681b0a61d61a6c8f0c4

Implemented as discussed on Slack, the **Legend** drops out in **FieldSetField** in favor of using a **Label** component and the `label` prop. We can silently swap to a **Legend** later if we need to.